### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by yriveiro** [Nicolab/atom-local-history#32]
**Atom Version**: 0.207.0
**System**: Mac OS X 10.10.3
**Thrown From**: [local-history](https://github.com/Nicolab/atom-local-history) package, v3.2.2

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/yriveiro/.atom/packages/local-history/lib/local-history-view.js:147

TypeError: undefined is not a function
  at LocalHistoryView.openDifftool (/Users/yriveiro/.atom/packages/local-history/lib/local-history-view.js:147:31)
  at LocalHistoryView.openDifftoolForCurrentFile (/Users/yriveiro/.atom/packages/local-history/lib/local-history-view.js:142:17)
  at LocalHistoryView.confirmed (/Users/yriveiro/.atom/packages/local-history/lib/local-history-view.js:102:10)
  at LocalHistoryView.module.exports.SelectListView.confirmSelection (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -4:53.7.0 docblockr:parse-tab (atom-text-editor.editor.cursor-block.vim-mode.is-focused.insert-mode)
     -4:53.7.0 editor:indent (atom-text-editor.editor.cursor-block.vim-mode.is-focused.insert-mode)
     -4:52.7.0 core:paste (atom-text-editor.editor.cursor-block.vim-mode.is-focused.insert-mode)
     -4:50.4.0 editor:move-to-first-character-of-line (atom-text-editor.editor.cursor-block.vim-mode.is-focused.insert-mode)
     -4:46.2.0 vim-mode:activate-command-mode (atom-text-editor.editor.cursor-block.vim-mode.is-focused.insert-mode.autocomplete-active)
     -4:44.2.0 vim-mode:delete-right (atom-text-editor.editor.cursor-block.vim-mode.is-focused.command-mode)
     -4:43.8.0 vim-mode:repeat (atom-text-editor.editor.cursor-block.vim-mode.is-focused.command-mode)
     -4:43.3.0 core:save (atom-text-editor.editor.cursor-block.vim-mode.is-focused.command-mode)
     -0:14.9.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.cursor-block.vim-mode.command-mode.is-focused)
     -0:14.7.0 core:paste (atom-text-editor.editor.mini.is-focused)
     -0:14 editor:move-to-first-character-of-line (atom-text-editor.editor.mini.is-focused)
 10x -0:13.5.0 core:delete (atom-text-editor.editor.mini.is-focused)
     -0:11.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:07.6.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:07.6.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:03.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "autocomplete",
      "meteor-api",
      "meteor-helper",
      "meteor-snippets"
    ],
    "themes": [
      "atom-dark-ui",
      "seti-syntax"
    ]
  },
  "local-history": {}
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.27.12
autocomplete-paths, v1.0.2
autocomplete-php, v0.3.6
autocomplete-phpunit, v1.0.2
block-cursor, v0.12.5
color-picker, v2.0.4
docblockr, v0.7.2
file-icons, v1.5.7
flex-tool-bar, v0.4.2
git-tab-status, v1.9.2
highlight-line, v0.10.2
linter, v0.12.6
linter-php, v0.0.15
linter-shellcheck, v0.0.8
local-history, v3.2.2
markdown-format, v2.5.1
minimap, v4.9.4
minimap-git-diff, v4.1.3
php-debug, v0.1.1
phpunit, v1.1.1
seti-syntax, v0.4.0
tool-bar, v0.1.6
travis-ci-status, v0.15.0
vim-mode, v0.51.0
wakatime, v4.0.9

# Dev
No dev packages
```

**Comment by JohnMurga** [comment]
Fixed by my pull request, still awaiting review

**Comment by Nicolab** [comment]
Fixed by #31, thanks @JohnMurga :wink:
